src/nvim/menu.c

```diff
@@ -679,7 +677,9 @@ static dict_T *menu_get_recursive(const vimmenu_T *menu, int modes)
   tv_dict_add_nr(dict, S_LEN("hidden"), menu_is_hidden(menu->dname));
 
   if (menu->mnemonic) {
-    mnemonic_len = utf_char2bytes(menu->mnemonic, (u_char *)buf);
+    int mnemonic_len;
+    char_u buf[8];  // > max value of char2bytes
```

Comment:
We typically use `0`.  It's the same as `'\0'`.

I specifically chose `char` instead of `char_u`.  You'll need to cast one way or the other, but we've been trending towards use of `char` (as you'll note in `tv_dict_add_str()`'s signature).